docs/serving/installing-istio.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Performing a Custom Istio Installation"
+weight: 15
+type: "docs"
+---
+
+Use this guide to perform a custom installation of Istio for use with Knative.
+
+## Download Istio
+
+Run below command to download Istio.
+```shell
+# Download and unpack Istio
```

Comment:
This all seems a little complicated; is there a reason not to use the command istio suggests:

```
curl -L https://git.io/getLatestIstio | ISTIO_VERSION=1.1.5 sh -
```
From: https://istio.io/docs/setup/kubernetes/download/